src/runner.js

```diff
@@ -21,7 +21,9 @@ const speedoUserAgent = `webdriver/${webdriverPkg.version} (Speedo/${speedoPkg.v
  * @param  {String} logDir     path to directory to store logs
  * @return {Object}            containing result and detail information of performance test
  */
-export default async function runPerformanceTest(username, accessKey, argv, name, build, logDir, retryCnt = 0) {
+export default async function runPerformanceTest(
+    username, accessKey, argv, name, build, logDir, retryCnt = 0, checkJankiness = false
```

Comment:
Let's keep the `retryCnt` param at the end
```suggestion
    username, accessKey, argv, name, build, logDir, checkJankiness, retryCnt = 0
```

and call it as follows:

```js
const retriedResult = await runPerformanceTest(username, accessKey, config, jobName, buildName, logDir, jankinessScore)